py/src/test_call.py

```diff
@@ -413,16 +538,96 @@ def test_fee_estimate_on_positive_directly():
     }
 
 
-def test_fee_estimate_on_positive():
+def test_fee_estimate_for_declare_transaction_directly():
     # fee estimation is a new thing on top of a call, but returning only the estimated fee
```

Comment:
This comment is repeated in estimate_fee tests but iiuc it's no longer valid.

Reply:
Thanks, I've removed all those comments.